index.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<html lang="English">
```

Comment:
The value of the `lang` attribute should be a language code, not a full word such as "English." http://www.w3.org/International/questions/qa-html-language-declarations